install.sh

```diff
@@ -22,86 +92,304 @@ deps=(
 "librtaudio6"
 "libkf5bluezqt-dev"
 "libtag1-dev"
+"qml-module-qtquick2"
+#TODO delete below after test
+#"qml-module-qtquick*"
+"libglib2.0-dev"
+"libgstreamer1.0-dev"
+"gstreamer1.0-plugins-base-apps"
+"gstreamer1.0-plugins-bad"
+"gstreamer1.0-libav"
+"gstreamer1.0-alsa"
+"libgstreamer-plugins-base1.0-dev"
+"qtdeclarative5-dev"
+"qt5-default"
+"libgstreamer-plugins-bad1.0-dev"
+"libunwind-dev"
+"qml-module-qtmultimedia"
 )
 
-#loop through dependencies and install
-for app in ${deps[@]}; do
-	echo "installing: " $app
-	sudo apt -qq -o=Dpkg::Use-Pty=0 install $app -y > /dev/null 2> /dev/null
-
-	if [[ $? > 0 ]]
-	then
-	    echo $app " Failed to install, quitting"
-	    exit
-	else
-	    echo $app " Installed ok"
-	    echo
-	    
-	fi
-done
-
-echo "All dependencies installed"
-echo
-
-#make ilclient
-echo "making ilclient"
-make /opt/vc/src/hello_pi/libs/ilclient
-if [[ $? > 0 ]]
+###############################  dependencies  #########################
+if [ $deps = false ]
   then
-    echo "unable to make ilclient"
-  exit
-else
-  echo "made ok"
-  echo
+    echo skipping dependencies '\n'
+  else
+    #loop through dependencies and install
+    echo installing dependencies
+    echo Running apt update
+    sudo apt update
+
+    installString="sudo apt install "
+
+    #create apt install string
+    for i in ${dependencies[@]}; do
+      installString+=" $i"
+    done
+
+    #run install
+    echo ${installString}
+    ${installString}
+    if [[ $? -eq 0 ]]; then
+        echo -e All dependencies Installed ok '\n'
+    else
+        echo Package failed to install with error code $?, quitting check logs above
+        exit
+    fi
 fi
 
-#begin cmake
-echo "beginning cmake for raspberry pi"
-cmake -DRPI_BUILD=TRUE -DGST_BUILD=TRUE -DCMAKE_BUILD_TYPE=Release . 
-if [[ $? > 0 ]]
-  then
-    echo "Cmake error"
-  exit
+
+###############################  AASDK #########################
+if [ $aasdk = false ]; then
+	echo -e Skipping aasdk '\n'
 else
-  echo "Cmake ok"
-  echo #####
-fi
+  #change to parent directory
+  cd ..
 
-#start make
-echo "beginning make, this will take a while"
-make
-if [[ $? > 0 ]]
-  then
-    echo "make error check output above"
+  #clone aasdk
+  git clone $aasdkRepo
+  if [[ $? -eq 0 ]]; then
+    echo -e Aasdk Cloned ok '\n'
+  else
+    cd aasdk
+    if [[ $? -eq 0 ]]; then
+      git pull $aasdkRepo
```

Comment:
Please add check if command was successful, as script continues in case of error: 
```
pi@raspberrypi:~/dash $ ./install.sh --aasdk
skipping dependencies
fatal: destination path 'aasdk' already exists and is not an empty directory.
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Total 4 (delta 3), reused 3 (delta 3), pack-reused 0
Unpacking objects: 100% (4/4), done.
From https://github.com/OpenDsh/aasdk
 * branch            HEAD       -> FETCH_HEAD
Updating 9170cef..7cb2b41
error: Your local changes to the following files would be overwritten by merge:
        aasdk_proto/ButtonCodeEnum.proto
Please commit your changes or stash them before you merge.
Aborting
cloned OK

```